code-reviews/README.md

```diff
@@ -2,69 +2,54 @@
 
-### General Guidance
+To ensure that the code review process is healthy and meets the goals stated above, consider following these guidelines:
 
 1. Although modern DevOps environments incorporate tools for managing PRs, it can be useful to label tasks pending for review or to have a dedicated place for them on the task board.
     * AzDO: [Customize cards](https://docs.microsoft.com/en-us/azure/devops/boards/boards/customize-cards?view=azure-devops)
     * AzDO: [Add columns on task board](https://docs.microsoft.com/en-us/azure/devops/boards/sprints/customize-taskboard?view=azure-devops#add-columns)
 2. In the daily standup meeting check tasks pending for review and make sure they have reviewers assigned.
 3. Junior teams and teams new to the process can consider creating separate tasks for reviews together with the tasks themselves.
 
-### Size guidance
-
-We should always aim to have Pull Requests be as small as possible, without losing context and technical feasibility. Small pull requests have multiple advantages:
-
-1. It's easier to review; a clear benefit for the reviewers.
-2. It's easier to deploy, at least in terms of a bigger Pull Request. This is aligned with the strategy of release fast and release often.
-3. Minimizes possible conflicts and stale Pull Requests, which are difficult to merge and keep in sync with master either because they're very dynamic or contain refactoring.
-
-However, we should be aware to avoid having Pull Requests that include code without context or very loose coupled.
-
-There are times where seem a big Pull Request is unavoidable, however, there are some strategies to keep Pull Requests small depending on the "cause" of the ineluctability:
-
-#### Minimum Working Components
+As an author of a PR, you should:
 
-#### Layers
+1. Add relevant reviewers:
+    * Add one or more reviewers (depending on your project's guidelines) to the PR. Ideally, you would add at least someone who has expertise and is familiar with the project or the language used
+    * Adding someone less familiar with the project or the language can aid in verifying the changes are understandable, easy to read, and increases the expertise within the team
+    * In CSE code-with projects with a customer team, it is important to include reviewers from both organizations for knowledge transfer
+      * In AzDO this can be enforced by setting groups as required reviewers, see [Automatically include code reviewers](https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops#automatically-include-code-reviewers)
+1. Be open to receive feedback, discuss design/code logic and address all comments as follows:
+   * Resolve a comment, if the requested change has been made.
+   * Mark the comment as "won't fix", if you are not going to make the requested changes and provide a clear reasoning - "I'll do it later" is not an acceptible reason!
```

Comment:
If there is a comment out of scope of the current review / work item, we could add that adding a task for it and resolving as fix later would be an option.

Reply:
That is true, although it's a faux pas by reviewer. Will add.

Reply:
Fixed